aait/backend/group-2B/Application.Tests/Features/Authentication/Handlers/Queries/LoginUserRequestHandlerTest.cs

```diff
@@ -0,0 +1,154 @@
+using AutoMapper;
+using HRLeaveManagement.Application.Profiles;
+using Microsoft.Extensions.Options;
+using Moq;
+using Shouldly;
+using SocialSync.Application.Common.Responses;
+using SocialSync.Application.Contracts.Infrastructure;
+using SocialSync.Application.Contracts.Persistence;
+using SocialSync.Application.DTOs.Authentication;
+using SocialSync.Application.Features.Authentication.Handlers.Queries;
+using SocialSync.Application.Features.Authentication.Requests.Commands;
+using SocialSync.Application.Features.Authentication.Requests.Queries;
+using SocialSync.Application.Tests.Mocks;
+using SocialSync.Infrastructure.DateTimeService;
+using SocialSync.Infrastructure.JWT;
+using SocialSync.Infrastructure.PasswordService;
+
+namespace SocialSync.Application.Tests.Features.Authentication.Handlers.Commands;
+
+public class LoginUserRequestHandlerTest
+{
+    private readonly IMapper _mapper;
+    private readonly Mock<IUnitOfWork> _mockUnitOfWork;
+    private readonly RegisterUserCommandHandler _registerHandler;
```

Comment:
You don't need to register the user to test the loginUserRequestHandler. You can hardcoded the user in usermock repository

Reply:
Noted.